pkg/telemetry/telemetry.go

```diff
@@ -41,7 +42,12 @@ const (
 	podCount                 = "pod_count"
 	helmManagedResourceCount = "helm_resource_count"
 
-	timestampFieldName = "timestamp"
+	timestampFieldName          = "timestamp"
+	stackMonitoringLogsCount    = "stack_monitoring_logs_count"
+	stackMonitoringMetricsCount = "stack_monitoring_metrics_count"
+	serviceCount                = "service_count"
+	pipelinesCount              = "pipelines_count"
+	pipelinesRefCount           = "pipelines_ref_count"
```

Comment:
```suggestion
	stackMonitoringLogsCount    = "stack_monitoring_logs_count"
	stackMonitoringMetricsCount = "stack_monitoring_metrics_count"
	serviceCount                = "service_count"
	pipelineCount              = "pipeline_count"
	pipelineRefCount           = "pipeline_ref_count"
```

No strong opinion on this but we started to use singular for almost all `xxx_count`.

Nit: Move these new vars just under `helmManagedResourceCount`.

To go further, I'd keep these constants local in the `logstashStats` func waiting to improve more globally the telemetry (check my other comment). 
It's not very clean at the moment, things are not reusable (this is historycal tech debt not from your PR) and for example now we end up with a duplicate `stack_monitoring_logs_count:

https://github.com/elastic/cloud-on-k8s/blob/beaa45767c2c22179f6e318705a5395bbf4a7779/pkg/telemetry/telemetry.go#L232